ja/core-libraries/caching.rst

```diff
@@ -1,20 +1,583 @@
-    `Github <https://github.com/cakephp/docs>`_ or use the **Improve This Doc**
-    button to directly propose your changes.
+    CakePHP ORMで作成したクエリの結果をキャッシュする場合は、 :ref:`caching-query-results` セクションで説明しているように、
+    Queryオブジェクトのビルトインキャッシュ機能を使用する方が良いです。
+
+一度に複数のキーを書き込む
+--------------------------
+
+.. php:staticmethod:: writeMany($data, $config = 'default')
+
+一度に複数のキャッシュキーを書き込む必要が出るかもしれません。
+``write()`` を複数回呼び出すこともできますが、 ``writeMany()`` はCakePHPがより効率的なストレージAPIを使用できるようにします。
+例えば、 ``writeMany()`` を使用すると、Memcachedを使用するときに複数のネットワーク接続を保存できます。::
+
+    $result = Cache::writeMany([
+        'article-' . $slug => $article,
+        'article-' . $slug . '-comments' => $comments
+    ]);
+
+    // $resultは以下を含みます
+    ['article-first-post' => true, 'article-first-post-comments' => true]
+
+Read-through キャッシュ
+-----------------------
+
+.. php:staticmethod:: remember($key, $callable, $config = 'default')
+
+Cacheを使用すると、Read-throughキャッシュを簡単に行うことができます。
+指定されたキャッシュキーが存在する場合、それが返されます。
+キーが存在しない場合、呼び出し可能オブジェクトが呼び出され、結果がキャッシュに格納されます。
+
+たとえば、リモートサービスコールの結果をキャッシュすることがよくあります。
+あなたはこれをシンプルにするために ``remember()`` を使うことができます::
+
+    class IssueService
+    {
+
+        public function allIssues($repo)
+        {
+            return Cache::remember($repo . '-issues', function () use ($repo) {
+                return $this->fetchAll($repo);
+            });
+        }
+
+    }
+
+
+キャッシュからの読み込み
+========================
+
+.. php:staticmethod:: read($key, $config = 'default')
+
+``Cache::read()`` は、``$key`` 配下に格納されたキャッシュされた値を ``$config`` から読み込むために使用されます。
+``$config`` がnullの場合、デフォルトの設定が使用されます。
+``Cache::read()`` は、有効なキャッシュであればキャッシュされた値を返し、キャッシュが期限切れになっているか存在しない場合は ``false`` を返します。
+キャッシュの内容はfalseと評価される可能性があるので、必ず厳密な比較演算子 ``===`` または ``!==`` を使用してください。
+
+例::
+
+    $cloud = Cache::read('cloud');
+
+    if ($cloud !== false) {
+        return $cloud;
+    }
+
+    // クラウドデータを生成する
+    // ...
+
+    // キャッシュにデータを保存する
+    Cache::write('cloud', $cloud);
+    return $cloud;
+
+一度に複数のキーを読み込む
+--------------------------
+
+.. php:staticmethod:: readMany($keys, $config = 'default')
+
+一度に複数のキーを書き込んだ後、あなたは恐らくそれらを同様に読み込みたいでしょう。
+``read()`` を複数回呼び出すこともできますが、``readMany()`` はCakePHPがより効率的なストレージAPIを使用できるようにします。
+例えばMemcachedを使用している場合、 ``readMany()`` を使用して、複数のネットワーク接続を保存できます。::
+
+    $result = Cache::readMany([
+        'article-' . $slug,
+        'article-' . $slug . '-comments'
+    ]);
+    // $result は以下を含みます
+    ['article-first-post' => '...', 'article-first-post-comments' => '...']
+
+
+キャッシュからの削除
+====================
+
+.. php:staticmethod:: delete($key, $config = 'default')
+
+``Cache::delete()`` を使うと、キャッシュされたオブジェクトをストアから完全に削除できます。::
+
+    // キーの削除
+    Cache::delete('my_key');
+
+一度に複数のキーの削除
+----------------------
+
+.. php:staticmethod:: deleteMany($keys, $config = 'default')
+
+一度に複数のキーを書き込んだら、それらを削除したいかもしれません。
+``delete()`` を複数回呼び出すこともできますが、``deleteMany()`` はCakePHPがより効率的なストレージAPIを使用できるようにします。
+例えば、 ``deleteMany()`` を使用して、Memcachedを使用するときに複数のネットワーク接続を保存できます。::
```

Comment:
I asked and I understand what it means now. They say that it means `minimizes` here and I think It could be translated into Japanese as `防ぐ` here. If you call `delete()` twice, two delete operations will be sent over the network connections. But if you use `deleteMany()`, one delete operations will be sent. Therefore, it says `save multiple network connections`.